### PR TITLE
LPS-76785 Rename

### PR DESCRIPTION
**Pull request by pedroqueiroz94**

### CODE REVIEW

**Comment by pedroqueiroz94** [comment]
ci:test

**Comment by leoadb** [comment]
@pedroqueiroz94 We need to rename the file names too. Thanks!

**Comment by pedroqueiroz94** [comment]
@leoadb , that should be good now!

**Comment by leoadb** [comment]
@pedroqueiroz94 Did you retest after rename? :)

**Comment by pedroqueiroz94** [comment]
@leoadb yeah, I tested manually. :) 
I'm not sure we have poshi tests to catch any issue on this PR, that's why I didn't re-run ci:retest.

File: dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_element_set_descriptive.jsp [hunk 1]
```diff
@@ -30,22 +30,22 @@ String href = (String)request.getAttribute(WebKeys.SEARCH_ENTRY_HREF);
 
 <div class="clamp-container">
 	<h4 class="truncate-text">
-		<aui:a cssClass="record-set-name" href="<%= href %>">
+		<aui:a cssClass="form-instance-name" href="<%= href %>">
```
Comment (leoadb) [inline dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_element_set_descriptive.jsp]: @pedroqueiroz94 You've changed the class name. What about the css file?

Comment (pedroqueiroz94) [reply dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_element_set_descriptive.jsp]: @leoadb There's no reference to `record-set-name` nor `form-instance-name` in any CSS files. Should we remove this class or leave it this way?

Comment (leoadb) [reply dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_element_set_descriptive.jsp]: Weird :) ... leave it as is

**Comment by leoadb** [comment]
ci:retest

**Comment by leoadb** [comment]
@pedroqueiroz94 Can you please rebase?

**Comment by pedroqueiroz94** [comment]
ci:retest

**Pull request merged**